src/material/chips/chip-row.ts

```diff
@@ -88,6 +88,9 @@ export class MatChipRow extends MatChip implements AfterViewInit {
 
   @Input() editable: boolean = false;
 
+  /** Function that maps option's control value to its display value. Used in case of editable="true" and having an object as value */
+  @Input() displayWith: ((value: any) => string) | null = null;
```

Comment:
We can have an input `getByProperty` (or some other name) which accepts a property name as a string. And inside `matChipEditInput` we can check if getByProperty is present, access the value by it, otherwise - we have a string value and we keep the default behavior. Your thoughts?

Reply:
@crisbeto can you please provide input here? I want to know if it is a good approach before doing a commit and then refactor it